configs/llm-router/aws.yaml

```diff
@@ -0,0 +1,7 @@
+head_node_type:
```

Comment:
you don't need these anymore

Reply:
yeah but since you merged into the existing template, you don't need new compute configs at all

Reply:
ok let me remove those files then

Reply:
@akshay-anyscale I am not sure what to do about the landing it in the product repo. There I need to specify configs, see e.g. https://github.com/anyscale/product/blob/master/backend/workspace-templates.yaml#L84 and I don't think the configs here: https://github.com/anyscale/product/blob/master/backend/workspace-templates.yaml#L246C12-L246C43 would work

Reply:
you shouldn't have to make a product repo change for this since the files are in the existing template. Is the only gap that for GCE it doesn't have the serverless config? @kouroshHakha why is that the case?

Reply:
why not make use of basic-server-less configs everywhere? 
```
head_node_type:
  name: head
  instance_type: n1-standard-8
worker_node_types: []
auto_select_worker_config: true
```

Reply:
I don't know why llm finetuning is not on serverless for gce. Maybe that has slipped during transition for some reason? I haven't noticed that until now.

Reply:
I think @anmscale mentioned a GPU head node was a hard req for this workspace; if that has changed, yes, please, let's use serverless :)